kubernetes/process_manifest.rb

```diff
@@ -0,0 +1,132 @@
+require "yaml"
+require "fileutils"
+
+if ARGV.count != 3
+  puts "usage: #{$PROGRAM_NAME} container-manifests-dir velum-source-code-dir salt-dir"
+  Process.exit 1
+end
+
+CONTAINERS_MANIFESTS_DIR = ARGV[0]
+VELUM_SOURCE_CODE_DIR = ARGV[1]
+SALT_DIR = ARGV[2]
+
+CONTAINERS_MANIFESTS_ORIG_DIR = "/usr/share/caasp-container-manifests".freeze
+SALT_ORIG_DIR = "/usr/share/salt/kubernetes".freeze
+
+def container_volume(name:, path:)
+  {
+    "name"      => volume_name(name: name),
+    "mountPath" => path
+  }
+end
+
+def patch_container_image(container)
+  if container["image"] =~ /^sles12\/velum/
+    container["image"] = "sles12/velum:development"
+  elsif container["image"] =~ /^sles12/
+    container["image"] = "docker-testing-registry.suse.de/#{container["image"]}"
+  end
```

Comment:
Just to play safe, could you add an `else` case, with an error message or something. This could help out when debugging 😄